text/src/autogluon/text/automm/data/process_text.py

```diff
@@ -320,6 +322,48 @@ def get_trimmed_lengths(
         else:
             return np.minimum(lengths, max_length)
 
+    def save(
+            self,
+            path: str,
+    ):
+        """
+        Save the tokenizer and text processor.
+
+        Parameters
+        ----------
+        path
+            Saving path.
+        """
+        self.tokenizer.save_pretrained(path)
+        self.tokenizer = None
+        with open(os.path.join(path, "text_processor.pkl"), "wb") as fp:
+            pickle.dump(self, fp)
```

Comment:
need to restore the tokenizer so the logic won't impact the inner states of this object.

```python
tokenizer = self.tokenizer
self.tokenizer = None
# DO_SOMETHING
self.tokenizer = tokenizer

```